lib/ansible/modules/packaging/os/satellite_subscription.py

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Luc Stroobant (luc.stroobant@wdc.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+
+DOCUMENTATION = """
+---
+module: satellite_subscription
+short_description: Red Hat Satellite subcription (license) mangement module
```

Comment:
Typos here, but we tend to make short_descriptions different.
```
short_description: Red Hat Satellite subscription (license) management module
```
```suggestion
short_description: Manage host subscriptions (licenses) on Red Hat Satellite
```